src/components/FormFieldFile.jsx

```diff
@@ -14,6 +14,7 @@ const FormFieldFile = ({
   onInlineButtonClick,
```

Comment:
We need to add the MediaModal for uploading and selecting a file, like with PR #197

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/184/commits/6fccca1561984c0dba8dde801bf2ef9c0edbda32